Makefile

```diff
@@ -215,13 +215,20 @@ endif
 check-docker-compose: $(PYTHON_BIN)
 	@PATH=$(PYTHON_BIN):$(PATH) ./script/check_docker_compose.sh $(BEATS_VERSION)
 
-.PHONY: format-package build-package
-format-package: $(ELASTICPACKAGE)
-	@(cd apmpackage/apm; $(ELASTICPACKAGE) format)
+.PHONY: build-package
 build-package: $(ELASTICPACKAGE)
-	@rm -fr ./build/packages/apm/* ./build/apmpackage
-	@$(GO) run ./apmpackage/cmd/genpackage -o ./build/apmpackage -version=$(APM_SERVER_VERSION)
-	@(cd ./build/apmpackage; $(ELASTICPACKAGE) build && $(ELASTICPACKAGE) check)
+	$(MAKE) build-package-with-version APM_SERVER_BUILD_VERSION=$(APM_SERVER_VERSION)
+
+.PHONY: build-package-snapshot
+build-package-snapshot: $(ELASTICPACKAGE)
+	$(MAKE) build-package-with-version APM_SERVER_BUILD_VERSION=$(APM_SERVER_VERSION)-beta-$(shell date +%s)
```

Comment:
```suggestion
	$(MAKE) build-package-with-version APM_SERVER_BUILD_VERSION=$(APM_SERVER_VERSION)-dev-$(shell date +%s)
```

Reply:
These are allowed prereleses: [code](https://github.com/elastic/package-spec/blob/main/code/go/internal/validator/semantic/validate_prerelease.go#L27-L30). Would you like me to replace it with something else? Do you prefer to introduce another one?

Reply:
Thanks for the pointer. I think "preview" would be close enough.

Reply:
Renamed to "preview".